src/MSALWrapper.Test/TokenFetcherPublicClientTest.cs

```diff
@@ -73,7 +75,7 @@ public void Setup()
              .AddTransient<TokenFetcherPublicClient>((provider) =>
              {
                  var logger = provider.GetService<ILogger<TokenFetcherPublicClient>>();
-                 return new TokenFetcherPublicClient(logger, ResourceId, ClientId, TenantId);
+                 return new TokenFetcherPublicClient(logger, ResourceId, ClientId, TenantId, promptHint: this.promptHint);

```

Comment:
Here I think we should build a prompt hint to pass in, so that is always has `AzureAuth CLI` at the beginning. Perhaps a helper method to construct our prompt hint, that we can call here. 

Something like
```C#
private const string PromptHintPrefix = "Azure AUth CLI";
private string GetPromptHint() {
  if (string.IsNullOrEmpty(this.PromptHint) {
    return PromptHintPrefix;
  }
  else
  {
    return $"{PromptHintPrefix: {this.PromptHint}";
  }
}
```

Reply:
Created a new issue #24